src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -148,13 +154,17 @@ public static class TagHelperBlockRewriter
                         symbolOffset = 0;
```

Comment:
Ahh good catch, I reversed the setting this time around to be set to 0 instead of 1 initially. :+1: